<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>companyandshippingrates</class>
 <widget class="QDialog" name="companyandshippingrates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="OrderLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>382</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Company Order History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewFeedbackButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>540</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View Cargo Owner
 Feedback</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcCompanyCommissionButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Calculate Company Commission</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="compOrderHistory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>471</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="viewFeedbackField">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="commissionField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="viewFeedbackButton_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View Driver
Feedback</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="viewFeedbackField_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargo Owner Feedback</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Driver Feedback</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
